SmartDeviceLink/SDLKeyboardProperties.h

```diff
@@ -53,6 +76,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, strong, nonatomic) NSString *autoCompleteText;
 
+/**
+ Allows an app to show a list of possible autocomplete suggestions as the user types
+
+ Optional, 1-100 items
```

Comment:
Should add the max string length as well

Reply:
Fixed